cfme/configure/access_control.py

```diff
@@ -226,17 +249,67 @@ def copy(self):
         assert view.is_displayed
         return new_user
 
-    def delete(self, cancel=True):
-        """ Delete existing user
-            Args:
-                cancel: Default value 'True', user will be deleted
-                        'False' - deletion of user will be canceled
+    def _delete_using_all_selection(self):
         """
-        view = navigate_to(self, 'Details')
-        view.configuration.item_select('Delete this User', handle_alert=cancel)
+        Delete existing user by selecting it from the access control group list
+
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
+                not having appropriate permissions OR delete is not allowed
+                for currently selected user
+        """
+        name_column = VersionPick({Version.lowest: "Name",
+            "5.4": "Full Name"}).pick(self.appliance.version)
+
+        view = navigate_to(User, 'All')
+        for userrow in view.table:
```

Comment:
In this case, we are looking for user on first page. And in case we have more than 20 users in table, rest of the pages will be skipped. it is better to use PaginationPane.find_row_on_pages